reStream.sh

```diff
@@ -3,17 +3,24 @@
 # Current reStream.sh version
 version="1.2.0"
 
+# video_filters links:
+# https://ffmpeg.org/doxygen/trunk/pixfmt_8h_source.html
+# https://ffmpeg.org/ffmpeg-filters.html#eq
+
+# This date is used to determine video_filters. (1696246154 == 2023-10-02 11:29:14.927044663 +0000)
+rm2_old_xochitl_date=1696246154
+
 # default values for arguments
-remarkable="${REMARKABLE_IP:-10.11.99.1}" # remarkable IP address
-landscape=true                            # rotate 90 degrees to the right
-output_path=-                             # display output through ffplay
-format=-                                  # automatic output format
-webcam=false                              # not to a webcam
-hflip=false                               # horizontal flip webcam
-measure_throughput=false                  # measure how fast data is being transferred
-window_title=reStream                     # stream window title is reStream
-video_filters=""                          # list of ffmpeg filters to apply
-unsecure_connection=false                 # Establish a unsecure connection that is faster
+remarkable="10.11.99.1"   # remarkable connected through USB
+landscape=false           # rotate 90 degrees to the right
```

Comment:
Why was this default changed?

Reply:
I forgot that I had made modified the script for my own program previously. I will update this in the next commit.